openupgradelib/openupgrade_merge_records.py

```diff
@@ -332,14 +332,19 @@ def _adjust_merged_values_orm(env, model_name, record_ids, target_record_id,
         elif field.type in ('binary', 'many2one'):
             op = op or 'merge'
             if op == 'merge':
+                l = [x for x in l if x]
                 if not getattr(target_record, field.name) and l and not \
                         vals.get(field.name):
                     vals[field.name] = l[:1]
     # Curate values that haven't changed
     new_vals = {}
     for f in vals:
-        if vals[f] != getattr(target_record, f):
-            new_vals[f] = vals[f]
+        if fields[f].type != 'many2many':
```

Comment:
This line is failing:

https://travis-ci.org/OCA/OpenUpgrade/builds/460391918#L7111

Reply:
my bad 🙁

Reply:
Fixed in https://github.com/OCA/openupgradelib/pull/132